rocketpy/plots/fluid_plots.py

```diff
@@ -23,6 +28,28 @@ def __init__(self, fluid):
 
         self.fluid = fluid
 
+    def density_function(self, lower=None, upper=None):
+        """Plots the density as a function of temperature in Kelvin
+        and Pressure in Pascal.
+
+        Parameters
+        ----------
+        lower: tuple
+            Lower range of the temperature and pressure interval. If None
+            default values are used.
+        upper: tuple
+            Upper range of the temperature and pressure interval. If None
+            default values are used.
+        """
+        if lower is None:
+            lower = (100, atm)
+        if upper is None:
+            upper = (zero_Celsius + 30, atm * 50)
```

Comment:
"zero_Celsius + 30", but the plot is in Kelvin. is it right?

+

Maybe adding the "Kelvin" and "Pascal" descriptions to the plot labels?